src/app.ts

```diff
@@ -311,10 +315,12 @@ function getName(path: string | undefined): string | undefined {
   }
   const parts = path.split('.');
 
-  if (parts.length > 0)
+  if (parts.length > 0) {
```

Comment:
Maybe replace it with `return parts.length > 0 ? _.last(parts) : undefined` ?

Reply:
Actually, looking at this, `getNamespace()` is only called once:

https://github.com/Maxim-Mazurok/google-api-typings-generator/blob/c809a2f304adde530437a6cc99d0b4ed0ded35af/src/app.ts#L580-L584

And that local `methods` variable is not referenced, so it seems this can be removed.